labs/src/main/scala/org/scalalabs/basic/lab02/CollectionExercise.scala

```diff
@@ -74,8 +85,8 @@ object CollectionExercise04 {
   /**
    * Calculate the length of the longest word in a list of sentences.
    * To keep it simple it's ok to use String.split to extract all words of a sentence.
-   */
+//   */
   def calcLengthLongestWord(lines: String*): Int = {
-    error("fix me")
+    lines.map(_.split("\\W+")).toList.flatten.sortWith(_.length > _.length).head.length
```

Comment:
This line might be long enough to be worth breaking into multiple lines and using intermediate variables :).